docs/adding-a-new-lgsl-code.md

```diff
@@ -18,4 +26,20 @@ Providing tiers can be:
 - `county/unitary`
 - `district/unitary`
 
-Once your pull request has been merged the codes will become active overnight.
+## Activate relevant service interactions
+You can either do this by creating a local transaction in Publisher, but this
+may show the transaction on the live site before the links are ready, so check
+with your product manager to make sure this is OK, or use a migration instead.
+
+See db/migrate/20190603131841_enable_service1788.rb for an example of the data
```

Comment:
It would be cool if this could be a link to the file.

Reply:
Apparently it can.